src/api/app/models/event/base.rb

```diff
@@ -251,7 +251,23 @@ def watchers
       project = ::Project.find_by_name(payload['project'])
       return [] if project.blank?
 
-      project.watched_projects.map(&:user)
+      project.watched_projects.map(&:user).concat(project.watched_items.map(&:user)).uniq
```

Comment:
The combination of the two lists is what we need? :thinking: 

Is not better to use `project.watched_items` when the feature flag is enabled and `project.watched_projects` when it is not enabled?

Reply:
the problem is user might miss out notifications when they join the beta program. Since we haven't migrated the watched projects from the non beta watchlist over to the new one its safer to simply use them both.

Reply:
we also didn't remove + add event subscriptions when a user switches from beta to non beta and the other way around, I would be consistent with this decision.